packages/fluentui/react-northstar/src/components/Tooltip/Tooltip.tsx

```diff
@@ -244,6 +244,10 @@ export const Tooltip: React.FC<TooltipProps> &
       _.invoke(triggerElement, 'props.onMouseLeave', e, ...args);
     },
   };
+  let calculatedOffset = offset;
+  if (!calculatedOffset) {
```

Comment:
```suggestion
  if (typeof calculatedOffset === 'undefined') {
```

Can use more strict check? Then it will be possible to pass `null` to reset default `offset`.